src/univariate/continuous/logitnormal.jl

```diff
@@ -0,0 +1,153 @@
+of a [`Normal`](@ref) variate: if ``X \\sim \\operatorname{Normal}(\\mu, \\sigma)`` then
+``\\exp(X) \\sim \\operatorname{LogitNormal}(\\mu,\\sigma)``. 
+The probability density function is
+
+```math
+f(x; \\mu, \\sigma) = \\frac{1}{x \\sqrt{2 \\pi \\sigma^2}}
+\\exp \\left( - \\frac{(\\logit(x) - \\mu)^2}{2 \\sigma^2} \\right),
+\\quad x > 0
+```
+
+where the logit-Function is
+```math
+logit(x) = \\ln(frac{x}{1-x})
+\\quad 0 < x > 1
+```
+
+```julia
+LogitNormal()          # Logit-normal distribution with zero logit-mean and unit scale
+LogitNormal(mu)        # Logit-normal distribution with logit-mean mu and unit scale
+LogitNormal(mu, sig)   # Logit-normal distribution with logit-mean mu and scale sig
+
+params(d)            # Get the parameters, i.e. (mu, sig)
+median(d)            # Get the median, i.e. logistic(mu)
+```
+
+The following properties have no analytical solution but numerical 
+approximations. In order to avoid package dependencies for 
+numerical optimization, they are currently not implemented.
+
+```julia
+mean(d)      
+var(d)      
+std(d)
+mode(d)
+```
+
+Similarly, skewness, kurtosis, and entropy are not implemented.
+
+External links
+
+* [Logit normal distribution on Wikipedia](https://en.wikipedia.org/wiki/Logit-normal_distribution)
+
+"""
+struct LogitNormal{T<:Real} <: ContinuousUnivariateDistribution
+    μ::T
+    σ::T
+
+    LogitNormal{T}(μ::T, σ::T) where {T} = (@check_args(LogitNormal, σ > zero(σ)); new{T}(μ, σ))
+end
+
+LogitNormal(μ::T, σ::T) where {T<:Real} = LogitNormal{T}(μ, σ)
+LogitNormal(μ::Real, σ::Real) = LogitNormal(promote(μ, σ)...)
+LogitNormal(μ::Integer, σ::Integer) = LogitNormal(Float64(μ), Float64(σ))
+LogitNormal(μ::Real) = LogitNormal(μ, convert(typeof(μ),1.0))
+LogitNormal() = LogitNormal(0.0, 1.0)
+
+# minimum and maximum not defined for logitnormal
+# but see https://github.com/JuliaStats/Distributions.jl/pull/457
+@distr_support LogitNormal 0.0 1.0
+#insupport(d::Union{D,Type{D}},x::Real) where {D<:LogitNormal} = 0.0 < x < 1.0
+
+
+#### Conversions
+convert(::Type{LogitNormal{T}}, μ::S, σ::S) where 
+  {T <: Real, S <: Real} = LogitNormal(T(μ), T(σ))
+convert(::Type{LogitNormal{T}}, d::LogitNormal{S}) where 
+  {T <: Real, S <: Real} = LogitNormal(T(d.μ), T(d.σ))
+
+#### Parameters
+
+params(d::LogitNormal) = (d.μ, d.σ)
+location(d::LogitNormal) = d.μ
+scale(d::LogitNormal) = d.σ
+@inline partype(d::LogitNormal{T}) where {T<:Real} = T
+
+#### Statistics
+
+# meanlogitx(d::LogitNormal) = d.μ
+# varlogitx(d::LogitNormal) = abs2(d.σ)
+# stdlogitx(d::LogitNormal) = d.σ
+
+# mean, mode, and variance 
+# moved to NormalTransforms because
+# they depend on the Optim package.
+
+# skewness, kurtosis, entropy
+# not implemented
+
+median(d::LogitNormal) = logistic(d.μ)
+
+#### Evalution
+
+#TODO check pd and logpdf
+function pdf(d::LogitNormal{T}, x::Real) where T<:Real 
+    if zero(x) < x < one(x) 
+        return normpdf(d.μ, d.σ, logit(x)) / x / (1-x)
```

Comment:
Does it numerically make sense to use `a/(b*c)` instead of `a/b/c`?

Reply:
a/b/c always bugs me, never sure in which order to take it